tests/contrib/elasticsearch/test.py

```diff
@@ -186,7 +208,7 @@ def test_elasticsearch(self):
         result = es.search(sort=['name:desc'], size=100,
                            body={"query":{"match_all":{}}}, **args)
 
-        assert len(result["hits"]) == 3, result
+        assert len(result["hits"]["hits"]) == 3, result
```

Comment:
I assume we were trying to test the number of results, if that's not the case I can revert this change.